scripts/prepare/bundle.ts

```diff
@@ -73,11 +73,15 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
     optimized,
   });
 
+  const nonPresetEntries = allEntries.filter((f) => !path.parse(f).name.includes('preset'));
```

Comment:
Add a comment or reference to the issue as to why this is happening?

Reply:
good suggestion! I'll do that now.